EnglishFilter/sections/antiadblock.txt

```diff
@@ -581,6 +581,9 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/82049
+lootlinks.*#%#//scriptlet("set-constant", "canRunAds", "true")
+@@||lootlinks.*/js/prebid-ads.js
```

Comment:
I guess the exception isn't needed when you use a scriptlet. Move it to the content blocker section.